rpcd/playbooks/roles/rpc_maas/tasks/raxmon_agent_install.yml

```diff
@@ -39,12 +48,20 @@
     mode: "0600"
 
 - name: Verify raxmon-entities-list and MaaS API authentication does not fail
-  command: "raxmon-entities-list"
+  shell: |
+    {% if maas_venv_enabled | bool %}
```

Comment:
Why are you using the activate script here? You could define `maas_bin` based on whether or not a venv is in use and then unconditionally call: `{{ maas_bin }}/raxmon-entities-list`